examples/APP_RESOURCE.ttl

```diff
@@ -21,28 +21,26 @@
     ids:representation [
         a ids:AppRepresentation ;
         ids:mediaType <https://www.iana.org/assignments/media-types/application/zip> ;
-        ids:instance <https://connector.fit.fraunhofer.de/apps/instance/42> ;
+        ids:appDestribtutionService <https://example.com> ;
+        ids:appRunetimeEnvironment "Docker" ;
         ids:dataAppInformation [
             a ids:SmartDataApp ;
             ids:appDocumentation "App-related human-readable documentation.";
+            ids:appEnvironmentVariables "$Env1 = environment variable 1, $Env2 = environment variable 2";
+            ids:appStorageConfiguration "1 Docker volume required, e.g. -v /data";
             ids:appEndpoint [
                 a ids:AppEndpoint;
                 ids:appEndpointInformation "I am an app endpoint. I do endpoint things.";
                 ids:appEndpointDocumentation <https://app.swaggerhub.com/apis/app/1337> ;
                 ids:appEndpointType idsc:INPUT_ENDPOINT ;
                 ids:appEndPointPort 5000 ;
                 ids:path "/input";
-                ids:appEndpointMediaType <https://www.iana.org/assignments/media-types/application/json>
+                ids:appEndpointMediaType <https://www.iana.org/assignments/media-types/application/json>;
+                ids:appEndpointProtocol "HTTP/1.1";
```

Comment:
Please evaluate once more, if we should use a string or URI here. The media type in the line just above this one is an ObjectProperty instead. Any choice is okay to me.

Reply:
we are using an URI/IRI here, and are declaring it as a `ids:IANAMediatype` further below (line 47).  Therefore obj. property + IRI should be correct AFAIK.
As you may remember, we have the "abstract" ids:Mediatype superclass with two subclasses `IANA Mediatype` and `Custom Mediatype`. If someone wants to annotate the mediatype, (s)he has to declare it as either a `CustomMediatype` or `IANAMediatype` otherweise it wont validate successfully against our SHACL rules. Therefore it is not legit to just use some IRI.